### PR TITLE
[appledoc] Shell escape headers and output

### LINKED ISSUE #306: Error 404 for DFImageManager
Hi! 

There is a 404 for [DFImageManager 0.0.18](http://cocoadocs.org/docsets/DFImageManager/0.0.18), which was pushed several hours ago. Docs re-parse didn't help.

P.S. Appreciate your work on Cocoadocs, everything was perfect for previous 17 versions. Thank so much.

### PR DESCRIPTION
**Pull request by kylef**
I'm sure I fixed this before? Have we had this bug all along or has it been reintroduced? /c @orta @segiddins 

Closes #306

Also note: Didn't test this manually and we have no test coverage.

### CODE REVIEW

**Comment by segiddins** [comment]
I believe Orta recently reverted half of one of my commits, which might've caused this. 

-Samuel E. Giddins

On Mar 2, 2015, at 5:25 PM, Kyle Fuller <notifications@github.com> wrote:

I'm sure I fixed this before? Have we had this bug all along or has it been reintroduced? /c @orta @segiddins

Closes #306

You can view, comment on, or merge this pull request online at:

  https://github.com/CocoaPods/cocoadocs.org/pull/307

Commit Summary

[appledoc] Shell escape headers and output
File Changes

M classes/docset_generator.rb (6)
Patch Links:

https://github.com/CocoaPods/cocoadocs.org/pull/307.patch
https://github.com/CocoaPods/cocoadocs.org/pull/307.diff

File: classes/docset_generator.rb [hunk 1]
```diff
@@ -52,8 +54,8 @@ def create_docset
 
       *guides.generate_array_for_appledoc,
 
-      "--output", to,                                        # where should we throw stuff
-      *headers
+      "--output", to.shellescape,                            # where should we throw stuff
+      *headers.map { |header| header.shellescape }
```
Comment (segiddins) [inline classes/docset_generator.rb]: `map(&:shellescape)`

Comment (kylef) [inline classes/docset_generator.rb]: Ahh, it was `:` I was missing.

```
[8] pry(main)> ['dsf', 'sdf'].map(&shellescape)
NameError: undefined local variable or method `shellescape' for main:Object
from (pry):8:in `__pry__'
[9] pry(main)> ['dsf', 'sdf'].map(&Shellwords.shellescape)
ArgumentError: wrong number of arguments (0 for 1)
from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/shellwords.rb:123:in `shellescape'
```

**Comment by orta** [comment]
Ah I see, yeah, something was broken :dancer:

**Pull request merged**